Make service catalog as a root chart (#1562)
kyma-project/kyma: resources/service-catalog/README.md

# Service Catalog

## Overview

The Service Catalog consists of the following sub-charts:

> - `catalog` - A Kubernetes Incubator project which provides a Kubernetes-native workflow for integrating with [Open Service Brokers](https://www.openservicebrokerapi.org/). For more information about the project, see the [official documentation](https://github.com/kubernetes-incubator/service-catalog).

Reviewer: ```suggestion
- `catalog`which is a Kubernetes Incubator project that provides a Kubernetes-native workflow for integration with the [Open Service Brokers API](https://www.openservicebrokerapi.org/). For more information about the project, see the [official documentation](https://github.com/kubernetes-incubator/service-catalog).
```

> - `etcd-stateful` - A store for data.

Reviewer: ```suggestion
- `etcd-stateful` which is a data store.
```

## Details

> To learn more about the Service Catalog operations and usage, see the [Overview](../../../../docs/service-catalog/docs/001-overview-service-catalog.md) document.

Reviewer: ```suggestion
To learn more about the Service Catalog operations and usage, see the [Overview](https://github.com/kyma-project/kyma/blob/master/docs/service-catalog/docs/001-overview-service-catalog.md) document.
```
